base/utils/data/trait.dictionary.csv

```diff
@@ -91,4 +91,6 @@
 "SPRTMX";"SPRTMX";"cm";""
 "DMAX";"DMAX";"degree days";""
 "DMIN";"DMIN";"degree days";""
-"leaf_longevity";"leaf longevity";"year";""
\ No newline at end of file
+"leaf_longevity";"leaf longevity";"year";""
+"frprod_perc_10";"fine root allocation percentage vertically in soils (every 10 cm)";"0-100";"frprod_perc_10"
```

Comment:
This strikes me as a poor (and too model specific) choice for a trait name. First, the 'every 10cm' part implies that there's not one allocation, but a whole host of allocation parameters that have a joint, sum to 100%, constraint. Second, I'd think about this from a data-driven perspective -- (A) if we wanted to store empirical root profile and root growth data by depth, how would we do that and what should we call it? (B) given that data, how do we translate those posteriors into the thing this specific model needs? Finally, I tend to find proportion data easier to work with than percent.